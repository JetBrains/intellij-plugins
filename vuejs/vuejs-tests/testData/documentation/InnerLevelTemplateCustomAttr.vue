<template>
  <template about="" sl<caret>ot="" class="">
  </template>
</template>