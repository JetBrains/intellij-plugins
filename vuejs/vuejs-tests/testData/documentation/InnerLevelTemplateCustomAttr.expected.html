<div class='definition'><pre>slot</pre></div>
<div class='content'>
<p><b>Prefer v-slot in 2.6.0+</b></p><p>Used on content inserted into child components to indicate which named slot the content belongs to.</p>
</div><table class='sections'>
<tr><td valign='top' class='section'><p>Deprecated</td>
<tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>vue@2.6.10</td>
</table>